bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java

```diff
@@ -96,6 +96,13 @@ public SnmpTargetHandler(Thing thing, SnmpService snmpService) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        if (target.getAddress() == null) {
```

Comment:
Any reason not combining those if statements?
```suggestion
        if (target.getAddress() == null && !renewTargetAddress()) {
```

Reply:
Done